### PR TITLE
feat: build without ts plugin & add build:types script

### PR DESCRIPTION
**Pull request by mleralec**

### CODE REVIEW

**Review by P1X3L — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -148,7 +149,7 @@
     "stylelint-processor-styled-components": "=1.9.0",
     "surge": "=0.21.3",
     "svgo": "^2.2.0",
-    "typescript": "^4.3.2",
+    "typescript": "^4.4.3",
```
Comment (guillaumewttj) [inline package.json:152]: Vu qu'on upgrade vers typescript 4.4, il faudrait upgrade prettier vers sa version compatible:
`"eslint-config-prettier": "^8.3.0"`
et
`"prettier": "^2.4.0",`

**Review by guillaumewttj — APPROVED** [review]
💪 🎉

File: package.json (hunk 1 above)
Comment (mleralec) [reply package.json:152]: @guillaumewttj we can keep this for an another PR ?

Comment (guillaumewttj) [reply package.json:152]: yes, im creating one based on this branch. We will merge it when this one is ok 🙂

Comment (guillaumewttj) [reply package.json:152]: All good in #920

**Pull request merged**